pointmatcher/ErrorMinimizers/PointToGaussian.h

```diff
@@ -84,6 +85,7 @@ struct PointToGaussianErrorMinimizer: public PointToPlaneErrorMinimizer<T>
     //const T confidenceInPenalties;
     const bool force2D;
     const bool force4DOF;
+    const float noiseSensor;
```

Comment:
I hope I'm not being too picky. If that's the case feel free to disregard this comment

noiseSensor sounds like a noise sensor, a device that detects noise.

sensorNoise would indicate the expected noise of a sensor. I think it is a more accurate term.

Now, what is this quantity? The variance, standard deviation of the noise?